### ISSUE TITLE
Not able to train jpn_vert

### ISSUE THREAD

**Issue by kamui-7** [tesseract-ocr/tesseract#3161]
I followed the tutorial on the docs and tried finetuning jpn_vert. I added the font to lang_specific too. This is the output I get
```
Warning in pixScaleSmooth: ridiculously small scaling factor 0.010451
Image too small to scale!! (1x1 vs min width of 3)
Line cannot be recognized!!
Image not trainable
```
Here is the script I used
```sh
# generates training data
tesseract/src/training/tesstrain.sh \
        --fonts_dir ${FONTS_DIR} \
        --tessdata_dir  ${TESSDATA_DIR} \
        --langdata_dir ${LANG_DATA_DIR} \
        --linedata_only \
        --lang jpn \
        --fontlist "Tanuki Permanent Marker" \
        --noextract_font_properties \
        --output_dir train \
        --exposures "0"  

# extracts lstm from old traineddata
combine_tessdata -e ${TESSDATA_DIR}/jpn_vert.traineddata jpn.lstm

# finetunes model
lstmtraining \
    --continue_from jpn.lstm \
    --model_output output/jpn \
    --traineddata ${TESSDATA_DIR}/jpn_vert.traineddata \
    --train_listfile train/jpn.training_files.txt \
    --max_iterations 100

# combines into a .traineddata
lstmtraining --stop_training \
    --continue_from output/jpn_checkpoint \
    --traineddata ${TESSDATA_DIR}/jpn_vert.traineddata \
    --model_output output/jpn_vert.traineddata
```
What am I doing wrong here? It worked on jpn but did not work with jpn_vert.

**Comment by Shreeshrii** [comment]
See https://github.com/tesseract-ocr/docs/blob/master/MOCRadaptingtesseract2.pdf
section 3.1 about vertical text

**Comment by kamui-7** [comment]
I'm sorry, I still do not know how to apply the paper that you linked me to fix this problem. Are there any tutorials on training vertical japanese?

**Comment by Shreeshrii** [comment]
Closed via PR #3223

See discussion at https://github.com/tesseract-ocr/tesseract/issues/3001
